docs/variables/lastfm.md

```diff
@@ -0,0 +1,33 @@
+---
+id: lastfm
+---
+
+# $(lastfm)
+
+Returns the current playing song of specified [**last.fm**](https://www.last.fm/) profile.
+
+#### Parameters
+
+This variable takes **one** *required* parameter that is a **[last.fm](https://www.last.fm/) username** of who to fetch the current playing song of.
+
+#### Example Output
+
+* `$(lastfm aiden)`
```

Comment:
This actually supports custom formatting: ```$(lastfm aidenwallis {songname} by {artist})``` for example

Reply:
I'll add that as **Supported Placeholders**.